Azure/azure-sdk-for-js
# [DEBUG] [MGMT Libraries] Fixing build failures with "onboarding-to-rush/adding-tests/using-recorder"

Before moving the mgmt libraries to rush.. trying out adding tests and importing recorder.

- Navigate to `sdk/test-utils/recorder`
- Run `rush update`
- `rush build -t test-utils-recorder`
- `rushx pack`   `--- By this, you'd have ....recorder.tgz at test-utils/recorder folder---`
- Navigate to `sdk/compute/arm-compute`
- Run `npm install`
- Run `npm run build`
- Run `npm pack`  `---By this, you'd have ...compute.tgz at the arm-compute folder---`
- Navigate to `sdk/compute/arm-compute-v2`
- Run `npm install`
- Run `npm run build`


After the above steps, running `npm run integration-test:node` should result in the following.

<img src="https://user-images.githubusercontent.com/10452642/110051236-51c5e200-7d0a-11eb-94d4-8a0f6cb3f46c.png" height=400>
